lnschema_core/models.py

```diff
@@ -652,17 +652,19 @@ def df(
         join: str = "inner",
         limit: int = 100,
     ) -> pd.DataFrame:
-        """Convert to `pd.DataFrame`.
+        """Convert to :class:`pd.DataFrame`.
 
         By default, shows all direct fields, except `updated_at`.
 
-        If you'd like to include other fields, use parameter `include`.
+        Use parameter `include` to include other fields
```

Comment:
You forgot to put the period. 😆

Reply:
https://github.com/laminlabs/lnschema-core/commit/3d773c0c091bec3f916cafaaef7893e12bc7549b